lib/expression/evaluate.js

```diff
@@ -31,25 +36,41 @@ class Evaluator {
     }
     return Buffer.from(string, "utf8").toString("base64");
   }
-  html_decode([ arg ]) {
-    const string = this.execute(arg.type, arg);
-    if (!string) return "";
-
-    return ent.decode(string);
-  }
   digest_md5([ arg ]) {
     const string = this.execute(arg.type, arg);
+    // console.log(`digest_md5("${string}")`);
     if (!string) {
       return [];
     }
 
-    const md5 = crypto.createHash("md5").update(string).digest();
-    const esihash = [];
-    for (let offset = 0; offset < 16; offset += 4) {
-      esihash.push(md5.readInt32LE(offset));
-    }
+    // const crypto = require("crypto");
+    // const md5 = crypto.createHash("md5").update(string).digest();
+    // console.log("md5", md5);
+    // md5 <Buffer c6 8a 36 27 2a 71 4f 29 7a 4a be a5 15 82 a8 06>
+    // const esihash = [];
+    // for (let offset = 0; offset < 16; offset += 4) {
+    //   esihash.push(md5.readInt32LE(offset));
+    // }
+    // return esihash;
 
+    /* eslint-disable-next-line new-cap */
+    const md5 = CryptoJS.MD5(string);
```

Comment:
We can hopefully replace this dependency with our native implementation of MD5 digests `crypto.subtle.digest('MD5', arraybuffer-or-typedarray)`
https://js-compute-reference-docs.edgecompute.app/docs/globals/SubtleCrypto/prototype/digest#parameters